lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -242,9 +259,9 @@ def _run_subhandlers(self, request):
                 assert handler(request)
```

Comment:
this also seems like an abuse of assert; asserts should be used primarily to check pre and post conditions in a function.

A problem here is that the broad try/catch will catch the assertion error along with
all of the other errors that might happen, which we than re-raise as a SubhandlerError
and possibly lose the context of the original error.

I think this would be better to get ride of the try/catch and have something like:

```
if not handler(request):
    raise SubhandlerError('Failed to process request', request)
```